src/main/java/net/pms/dlna/RealFile.java

```diff
@@ -176,13 +173,24 @@ public synchronized void resolve() {
 				DLNAMediaDatabase database = PMS.get().getDatabase();
 
 				if (database != null) {
-					ArrayList<DLNAMediaInfo> medias = database.getData(fileName, file.lastModified());
-
-					if (medias != null && medias.size() == 1) {
-						setMedia(medias.get(0));
-						getMedia().finalize(getType(), input);
-						found = true;
+					ArrayList<DLNAMediaInfo> medias;
+					try {
+						medias = database.getData(fileName, file.lastModified());
+
+						if (medias != null && medias.size() == 1) {
+							setMedia(medias.get(0));
+							getMedia().postParse(getType(), input);
+							found = true;
+						}
+						//TODO: (Nad) Bug if more than one hit
```

Comment:
@SubJunk It's not a new bug in this PR, but I guess I was planning to try to handle it somehow when I wrote it. I must admit that somehow have overlooked it, I've been trying to go through all the ```//TODO: (Nad)```'s It's not a very serious bug though, it simply means that if multiple database entries are returned on one query (which shouldn't be possible), none will be used and the item will be re-parsed instead.